src/chapter17/EffectsAndTransformsDemo.java

```diff
@@ -0,0 +1,134 @@
+package chapter17;
+
+import javafx.application.*;  
+import javafx.scene.*;  
+import javafx.stage.*;  
+import javafx.scene.layout.*;  
+import javafx.scene.control.*;  
+import javafx.event.*;  
+import javafx.geometry.*;  
+import javafx.scene.transform.*; 
+import javafx.scene.effect.*; 
+  
+public class EffectsAndTransformsDemo extends Application {  
+  
+  double angle = 0.0; 
+  double scaleFactor = 0.4; 
+  double blurVal = 1.0; 
+ 
+  // Create initial effects and transforms. 
+  Reflection reflection = new Reflection(); 
+  BoxBlur blur = new BoxBlur(1.0, 1.0, 1); 
+  Rotate rotate = new Rotate(); 
+  Scale scale = new Scale(scaleFactor, scaleFactor); 
+ 
+  // Create push buttons.  
+  Button btnRotate = new Button("Rotate");  
+  Button btnBlur = new Button("Blur off"); 
+  Button btnScale = new Button("Scale"); 
+ 
+  Label reflect = new Label("Reflection Adds Visual Sparkle"); 
+  
+  // Blur the rotate button too
+  BoxBlur rotateBlur = new BoxBlur(5.0, 5.0, 2); 
+  boolean doBlur = true;
+ 
+  public static void main(String[] args) {  
+  
+    // Start the JavaFX application by calling launch().  
+    launch(args);    
+  }  
```

Comment:
The latest eclipse seems to have run configurations for JavaFX applications (so you wouldn't need `main`).